CHANGES

```diff
@@ -9,8 +9,8 @@
 
  Changes between 1.1.0g and 1.1.1 [xx XXX xxxx]
 
-  *) Added support for X448 and Ed448. Currently this is only supported in
-     libcrypto (not libssl). Heavily based on original work by Mike Hamburg.
+  *) Added support for X448 and Ed448 to both libcrypto and libssl. Heavily
```

Comment:
get rid of "to both librypto and libssl"

Reply:
Ok - done.